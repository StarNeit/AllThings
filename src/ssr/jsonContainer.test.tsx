import React from 'react'
import JsonContainer from './jsonContainer'
import render from 'react-test-renderer'

it('JsonContainer Component', () => {
  const tree = render.create(<JsonContainer id="test" json="{}" />).toJSON()
  expect(tree).toMatchSnapshot()
})
