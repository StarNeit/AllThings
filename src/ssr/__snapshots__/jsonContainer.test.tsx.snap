// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JsonContainer Component 1`] = `
<span
  data-state="{}"
  id="test"
  style={
    Object {
      "display": "none",
    }
  }
/>
`;
